app/src/main/java/com/github/swent/echo/viewmodels/HomeScreenViewModel.kt

```diff
@@ -99,7 +107,7 @@ constructor(
             fullChecked = false,
             from = 0f,
             to = 14f,
-            sortBy = SortBy.NONE
+            sortBy = null
```

Comment:
I think it doesn't really make sense that one can set the sortby to none, this would mean the list is not sorted when in reality it always is sorted the way we did it. This should probably be set to sortby.DATE_ASC to make sense in my opinion

Reply:
They are not sorted for now (check the events) but I agree it could be set to DATE_ASC. It's more a user convenience to see a dropDown named SortBy instead of DATE_ASC. I don't know if it's a big problem but I can change if you think it's necessary

Reply:
Are they not sorted by date automatically ? They should because that's how I did things

Reply:
I corrected everything you said above, thanks for the useful feedback !!